### PR TITLE
Fixed home screen styling

### PR DESCRIPTION
**Pull request by yuk0ga**
Fixed styling of home screen to match with the UI mockups. 
Haven't implemented a file picker yet though. 

The `primarySwatch` property in ThemeData only allowed material colors so I couldn't set the specific color used in the mockup, but I found a workaround [here](https://medium.com/flutterworld/flutter-material-color-conversion-ad1574f25828).

homescreen

### CODE REVIEW

File: lib/widgets/menuItem.dart [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class MenuItem extends StatelessWidget {
+  final String title;
+  final Widget trailing;
+
+  MenuItem({this.title, this.trailing});
```
Comment (kressnick25) [inline lib/widgets/menuItem.dart]: Add onTap callback function here to be passed to ListTile child

**Comment by kressnick25** [comment]
Styling looks good. Need to merge master into this branch/resolve merge conflicts and ensure that file selector still works before this can be merged.

File: lib/widgets/menuItem.dart (hunk 1 above)
Comment (yuk0ga) [reply lib/widgets/menuItem.dart]: Fixed in 3c9d7409084cc8fa7e55cdacd96c2b96dc5e4606

**Comment by yuk0ga** [comment]
- Updated flutter version for the workflow (1.17.5 -> 1.20.2)
- file selector works.

**Pull request merged**